defaults/main/template.yml

```diff
@@ -273,6 +271,13 @@ nginx_config_http_template:
         underscores_in_headers: false  # Boolean -- Not available in the 'location' context
         variables_hash_bucket_size: 64  # Available only in the 'http' context
         variables_hash_max_size: 1024  # Available only in the 'http' context
+      http2: # all options are for 'http' or 'server' contexts only unless noted otherwise
+        enable: false
+        body_preread_size: 64k
+        chunk_size: 8k # can be used in a 'location' context
```

Comment:
```suggestion
        chunk_size: 8k  # Not available in the 'location' context
```

Reply:
This change inverts the meaning of the comment. Chunk size is the only option that can be used in a location.

Reply:
It's easier to drop the comment, I guess. Along with the main one.